source/overloading.tex

```diff
@@ -3489,24 +3470,23 @@
 \indextext{overloading!unary operator}
 
 \pnum
-A prefix unary operator shall be implemented by a
-non-static member function\iref{class.mfct} with no parameters or a
-non-member function with one parameter.
+For a \grammarterm{unary-operator} \tcode{@},
```

Comment:
Should we use `\placeholder` for these `@`s?

Reply:
I think we have more @s for the three-way comparisons; if we do something in this area, we should do it in a separate effort.